server/utils/map.js

```diff
@@ -111,13 +111,18 @@ export function mapChallengeToLang(
     lang = 'en';
   }
   const translation = translations[lang] || {};
+  const isTranslated = (Object.keys(translation).length > 0);
```

Comment:
This doesn't work correctly when there are more than two languages.

We want to check if we are falling back to english because this challenge does not have the selected language translation.